### DISCUSSION TITLE
Is there a benefit of using my get for prereleases vs nuget?

### DISCUSSION THREAD

**Discussion by sjd2021** [elsa-workflows/elsa-core#3553] [Q&A] [answered]
I'm just trying to understand what benefit there is to using myget for prereleases and nuget for official releases. That's all I'm really trying to understand.

_Background information which isn't really relevant to answering this question - just how I came up with this thought:

Unfortunately, in my organization, we're locked into Azure and Nuget for our package configuration (another team controls this). In Azure, we have a custom feed that can inherit packages from Nuget.org. However, Azure doesn't currently allow you to support synchronization with multiple parent feeds. So there's not a good way for us to get MyGet packages added in. 
If the prereleases were available on Nuget.org, we'd be able to use the latest version of Elsa which contains the fixes we need ASAP like .NET 7 support._

**Comment by sfmskywalker** [comment, marked as answer]
We've been using MyGet for continuous builds from master. Because this generates a lot of package versions, it doesn't seem right to post them to NuGet.org, because then we cannot apply a retention policy (that I'm aware of at least).

The beauty of MyGet is that we can apply a retention policy that deletes older packages - as they should be, because they are transitory in nature and should not live on for eternity. Only releases (and as of recent, pre-releases too) should be deployed to NuGet. 

I would encourage the team you mentioned managing the package sources to allow pulling from the MyGet feed in order to be able to try out fixes and improvements.

**Reply by sjd2021** [reply]
"We've been using MyGet for continuous builds from master. Because this generates a lot of package versions"
"The beauty of MyGet is that we can apply a retention policy"

This makes sense. I thought someone had to specifically enqueue it, and I also didn't realize we couldn't set a retention policy on Nuget.org. Thanks for explaining this to me.

**Comment by sniper633** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
